flex_catalog/flex_catalog/views.py

```diff
@@ -30,7 +46,7 @@ def get_course_runs(self, request, slug=None):
         Forward the REST view to the get_course_runs in the object.
         """
         catalog = get_object_or_404(FlexibleCatalogModel.objects.select_subclasses(), slug=slug)
-        course_runs = catalog.get_course_runs()  # Call the model method
+        course_runs = catalog.get_catalog_items()  # Call the model method
         serializer = CourseOverviewBaseSerializer(course_runs, many=True)
```

Comment:
```suggestion
        serializer = self.get_serializer_class()(course_runs, many=True)
```

this is probaly needed

Reply:
same goes for the get search below. That was my bad originally